pyiron_atomistics/dft/job/generic.py

```diff
@@ -318,16 +318,6 @@ def set_kpoints(
             path_name=path_name,
         )
 
-    def calc_static(
-        self,
-        electronic_steps=100,
-        algorithm=None,
-        retain_charge_density=False,
-        retain_electrostatic_potential=False,
-    ):
-        self._generic_input["fix_symmetry"] = True
```

Comment:
Isn't it just this line that causes the problem discussed in the linked issues?  I rather like having `calc_static` on the generic DFT to give us a place where we can enforce generic parameters like `electronic_steps`.  (I suspect that the other three are not actually generic, they could be removed then.)

Reply:
Fine, I brought it back.

Reply:
Thanks for the additional clean up, too!